ciao-controller/compute_test.go

```diff
@@ -35,6 +35,21 @@ import (
 	"github.com/01org/ciao/testutil"
 )
 
+// ByTenantID is used to sort CNCI instances by Tenant ID.
```

Comment:
github doesn't let you leave comments on the commit message :-( But I think the commit message needs updating to reflect that the sorting is happening in the unit test.